spec/requests/api/v0/highlights_controller_spec.rb

```diff
@@ -480,6 +499,27 @@
           highlight2.source_id.to_sym => 1,
         })
       end
+
+      context 'colors passed as comma-separated values in the query parameters' do
+        it 'handles them fine' do
+          get "/api/v0/highlights/summary?source_type=openstax_page&scope_id=#{scope_1_id}&" \
+              "colors=yellow,pink"
```

Comment:
we note that the arrays are passed in as csv, and not as standard URL array syntax somewhere right?

Reply:
yeah, it is `key :collectionFormat, :csv` in the swagger